### PR TITLE
add module export plugin, add babel-polyfill to dependencies

### PR DESCRIPTION
**Pull request by siutsin**
Unable to install package from npm:

1. `babel-polyfill` not found
2. Crash for non-ES7 syntax
```js
var NaughtyChecker = require('NaughtyChecker');
var nc = new NaughtyChecker();

var fromOnline = function () {
  var result = nc.validate('naughty string').then(function (result) {
    console.log('ok');
  }).catch(function (error) {
    console.log('error');
  })
};
```

This PR fixed the above issues

### CODE REVIEW

**Review by gautamkrishnar — APPROVED** [review]

**Comment by gautamkrishnar** [comment]
@siutsin you can merge this when ready...

**Comment by siutsin** [comment]
OK, I will release 2.0.1 as well

**Comment by siutsin** [comment]
Oops, no push right to `develop` and `master`. Will need your help @gautamkrishnar, thanks.

**Comment by gautamkrishnar** [comment]
Just merged it to master. I will give push rights for you. I enabled it to prevent any accidental pushes using `--force`....

**Comment by siutsin** [comment]
Hmm... `2.0.1` deployed to npm for the commit `3601330` instead of any commit in `master`.
Should we deploy a `2.0.2` just to make it synchronised?

**Comment by gautamkrishnar** [comment]
Yes. I will do it myself. please wait, I just used a wrong version...

**Comment by siutsin** [comment]
OK, I will leave it to you, thanks.

**Comment by gautamkrishnar** [comment]
@siutsin released. Feel free to make any future releases. Please use github releases feature, and make necessary change logs here: https://github.com/gautamkrishnar/naughtychecker.js/releases

**Comment by siutsin** [comment]
Got it

**Pull request merged**
